### PR TITLE
DOC: remove copyright year

### PR DESCRIPTION
**Pull request by willschlitzer**
Updates the copyright year in `conf.py` to "2013-2024".

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def setup(app):
 
 # General information about the project.
 project = "GeoPandas"
-copyright = "2013–2022, GeoPandas developers"
+copyright = "2013–2024, GeoPandas developers"
```
Comment (martinfleis) [inline doc/source/conf.py:110]:
```suggestion
copyright = "2013–, GeoPandas developers"
```
This should solve it once and for all :)

Comment (jorisvandenbossche) [reply doc/source/conf.py:110]: Or something like `f"2013-{datetime.date.today().year}"` if we want to keep the year but automate it?

Comment (martinfleis) [reply doc/source/conf.py:110]: Fine with either.

Comment (m-richards) [reply doc/source/conf.py:110]: I think drop the year, that way there's no chance we get future issues/ pull requests indicating it's wrong (whereas we might if the generated docs from the last release have an old year).

As far as I understand it's not required anyway - https://hynek.me/til/copyright-years/

**Comment by martinfleis** [comment]
Thanks!

**Pull request merged**